presidio-analyzer/presidio_analyzer/predefined_recognizers/il_domain_recognizer.py

```diff
@@ -0,0 +1,22 @@
+import tldextract
+
+from presidio_analyzer import Pattern, PatternRecognizer
+
+# pylint: disable=line-too-long
+IL_DOMAIN_REGEX = u'(www.)?[\u05D0-\u05EA]{2,63}.co.il'  # noqa: E501'  # noqa: E501
```

Comment:
What about org.il, gov.il etc.?

Reply:
just an example for the POC... you are right